app/javascript/components/rooms/Rooms.jsx

```diff
@@ -6,7 +6,7 @@ import Recordings from '../recordings/Recordings';
 export default function Rooms() {
   return (
     <Row className="pt-5">
-      <Tabs defaultActiveKey="rooms">
+      <Tabs defaultActiveKey="rooms" unmountOnExit>
```

Comment:
For all tabs/tab that does not need to be linked to a route and uses a query the tab should be unmounted on exit.